server/src/apiClient/ShowsClient.js

```diff
@@ -0,0 +1,20 @@
+import got from "got";
+
+const TMDBShowApiKey = ""
```

Comment:
we need to load in this key from an env file 

Every member of the team will need to create a `.env` file in the root of server in order to load in an env key